sync_deps.py

```diff
@@ -7,7 +7,7 @@
 ### Update with: shark-workspace pin
 
 PINNED_VERSIONS = {
-  "iree": "06e05b3f86870cd5620df2f57ea8af4cb136c120",
+  "iree": "ff8072818e5996fbc9fb937ce21c8433bba001d5",
```

Comment:
I think you might want to keep the old hash as now you are not using the IREE tresleches branch?

Reply:
Yes, I suppose this is true.  And probably neither one is correct, as we want to pull in the latest optimizations.  I will rework the PR to pull in the HEAD of shared/tresleches-cpu.  Thanks for pointing this out!